otherlibs/stdune/src/path.mli

```diff
@@ -230,6 +236,37 @@ type t = private
 
 include Path_intf.S with type t := t
 
+module Table : sig
+  (** Specialized tables for path. We do implement all of [Hashtbl_intf.S] -
+      only what we use in dune. *)
+
+  type path := t
+
+  type 'a t
+
+  val create : int -> 'a t
+
+  val clear : 'a t -> unit
+
+  val mem : 'a t -> path -> bool [@@inline always]
```

Comment:
I'm getting 
```
the "inline" attribute cannot appear in this context
```
errors internally. Are you sure these attributes are placed correctly?

Reply:
Weird. It must be a difference between flambda and the standard compiler. I've moved the annotations to the .ml file like we do elsewhere.